### ISSUE TITLE
Rendering Crash?

### ISSUE THREAD

**Issue by Fortist** [thedarkcolour/Future-MC#3]
I had a barrel and threw some of the mod's stuff into it and got this crash
```
---- Minecraft Crash Report ----
// Surprise! Haha. Well, this is awkward.

Time: 2019-02-08 17:15:54 EST
Description: Rendering screen

java.lang.IllegalArgumentException: Cannot set property PropertyDirection{name=facing, clazz=class net.minecraft.util.EnumFacing, values=[north, south, west, east]} to down on block minecraftfuture:loom, it is not an allowed value
    at pl.asie.foamfix.common.FoamyBlockState.withProperty(FoamyBlockState.java:59)
    at com.herobrine.future.blocks.Loom.getStateFromMeta(Loom.java:50)
    at tyra314.toolprogression.handlers.TooltipEventHandler.onGetToolTipEvent(TooltipEventHandler.java:79)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_2880_TooltipEventHandler_onGetToolTipEvent_ItemTooltipEvent.invoke(.dynamic)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
    at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182)
    at net.minecraftforge.event.ForgeEventFactory.onItemTooltip(ForgeEventFactory.java:335)
    at net.minecraft.item.ItemStack.getTooltip(ItemStack.java:889)
    at net.minecraft.client.gui.GuiScreen.getItemToolTip(GuiScreen.java:143)
    at net.minecraft.client.gui.GuiScreen.renderToolTip(GuiScreen.java:137)
    at net.minecraft.client.gui.inventory.GuiContainer.renderHoveredToolTip(GuiContainer.java:175)
    at com.herobrine.future.blocks.tile.GuiBarrel.drawScreen(GuiBarrel.java:35)
    at net.minecraftforge.client.ForgeHooksClient.drawScreen(ForgeHooksClient.java:396)
    at net.minecraft.client.renderer.EntityRenderer.updateCameraAndRender(EntityRenderer.java:1124)
    at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1119)
    at net.minecraft.client.Minecraft.run(Minecraft.java:3942)
    at net.minecraft.client.main.Main.main(SourceFile:123)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196)
    at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231)
    at org.multimc.EntryPoint.listen(EntryPoint.java:143)
    at org.multimc.EntryPoint.main(EntryPoint.java:34)
```

**Comment by thedarkcolour** [comment]
What version of my mod is this?
What other mods are there? It looks like some stuff with foamfix.

**Comment by thedarkcolour** [comment]
Can you explain exactly what you did to cause this crash?

**Comment by thedarkcolour** [comment]
I think disabling 
`    B:smallPropertyStorage=true` in the foamfix.cfg file will fix it.

**Comment by thedarkcolour** [comment]
Fixed in 0.1.0

**Comment by thedarkcolour** [comment]
It seems I have not fixed the issue.

**Comment by thedarkcolour** [comment]
Fixed in 0.1.5.
